### PR TITLE
Formatted text export

### PR DESCRIPTION
**Pull request by ilyasgaraev**
https://aclgrc.atlassian.net/browse/PD-6344

### CODE REVIEW

**Comment by EmilShakirov** [comment]
Looks 🆗

File: lib/creek/cell_styles.rb [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        end
+      end
+    end
+
+    def [](index)
+      definitions[index]
+    end
+
+    def styles_xml
+      styles.styles_xml
```
Comment (KirillKayumov) [inline lib/creek/cell_styles.rb]: Can be delegated

File: lib/creek/html_extractor.rb [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    def html_from_xml(node)
+      str = ""
+      xml_elems = { strong: false, em: false, u: false }
+
+      node.children.each do |elem|
+        case elem.name
+        when "rPr"
+          elem.children.each do |property|
+            property_name = property.name.to_sym
+            xml_elems[EXCEL_STYLE_MAP[property_name]] = true if EXCEL_STYLE_MAP.keys.include?(property_name)
```
Comment (KirillKayumov) [inline lib/creek/html_extractor.rb]: Could use `#key?` method

**Comment by KirillKayumov** [comment]
👍

**Pull request merged**